src/test/java/reposense/model/RepoLocationTest.java

```diff
@@ -47,7 +47,8 @@ public void repoLocationParser_parseEmptyString_success() throws Exception {
     }
 
     /**
-     * Compares the information parsed by the RepoLocation model with the expected information
+     * Compares the information of {@code rawLocation} parsed by the RepoLocation model with {@code expectedRepoName}
+     * and {@code expectedOrganization}.
      */
     public void assertValidLocation(String rawLocation, String expectedRepoName,
             String expectedOrganization) throws Exception {
```

Comment:
Should move the `expectedOrganization` variable to line 52 and keep the `throws Exception` in line 53.